src/coreclr/inc/corcompile.h

```diff
@@ -56,6 +56,11 @@ inline ReadyToRunCrossModuleInlineFlags operator &( const ReadyToRunCrossModuleI
     return static_cast<ReadyToRunCrossModuleInlineFlags>(static_cast<uint32_t>(left) & static_cast<uint32_t>(right));
 }
 
+#ifdef TARGET_WASM
+// why was it defined only for x86 before?
```

Comment:
Because all other platforms have it defined in either the pal.h or in the Windows SDK.

Reply:
I don't see it in pal.h, so is it in the windows sdk then?